### PR TITLE
React capstone project

### PR DESCRIPTION
**Pull request by fahimy143**
### Project requirements

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.
Live Demo (https://stupendous-klepon-6b06f0.netlify.app)
Video Demo (https://www.loom.com/share/edd9b0424e4c4783b06775bbc328d8f9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ This is a react web app where users can look up and see details about countries
 
 ## Live Demo 
 
-[COMING SOON!]()
+[Country App](https://stupendous-klepon-6b06f0.netlify.app)
 
 ## Demo Video
 
-[Demo Video]()
+[Demo Video](https://www.loom.com/share/e73e6a5df5b2463db165293bcf33346f)
```
Comment (danmainah) [inline README.md]: - [ ] @fahimy143 great work in your presentation, however, you forgot to highlight the piece of code you found exciting or proud of. Also, you didn't mention the purpose of your project `eg countries application, help individuals find details about countries.

File: README.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ This is a react web app where users can look up and see details about countries
...
```
Comment (danmainah) [inline README.md:25]: - [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @fahimy143 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @fahimy143👋, 

### You did a marvelous job on the React capstone project and this is one of the best projects I have seen so far!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**